ahnlich/ai/src/error.rs

```diff
@@ -24,8 +24,8 @@ pub enum AIProxyError {
     UnexpectedDBResponse(String),
     #[error("Cannot Query Using Input. Store expects [{store_query_model_type}], but input type [{storeinput_type}] was provided")]
     StoreQueryTypeMismatchError {
-        store_query_model_type: AIStoreInputType,
-        storeinput_type: AIStoreInputType,
+        store_query_model_type: String,
```

Comment:
I think we can keep these as `AIStoreInputType` not `String`... so we know it can only happen with those types

Reply:
Yeah, I found that confusing. I changed to String because `StoreSetTypeMismatchError` uses String instead of AIStoreInputType, which is inconsistent for similar errors.

Reply:
Yeah points to the fact that I need to unify both really and switch them from Strings